packages/api-explorer/src/components/DocSdkUsage/DocSdkUsage.tsx

```diff
@@ -40,31 +48,59 @@ interface DocSdkUsageProps {
  *  links to the source files
  */
 export const DocSdkUsage: FC<DocSdkUsageProps> = ({ method }) => {
-  const tabs = useTabs()
   const { examples } = useContext(LodeContext)
   const languages = findExampleLanguages(examples, method.name)
   if (languages.length === 0) return <></>
 
+  const tableExamples = prepareExampleTableData(
+    languages,
+    examples,
+    method.operationId
+  )
+
   return (
     <CollapserCard id="examples" heading="SDK Examples">
-      <>
-        <TabList {...tabs}>
-          {languages.map((language) => (
-            <Tab key={language}>{language}</Tab>
-          ))}
-        </TabList>
-        <TabPanels {...tabs} pt="0">
-          {languages.map((language) => (
-            <TabPanel key={language}>
-              <DocExamples
-                lode={examples}
-                language={language}
-                operationId={method.operationId}
-              />
-            </TabPanel>
-          ))}
-        </TabPanels>
-      </>
+      <Card height="auto" p="medium" mt="small">
```

Comment:
Does this whole Card deserve a separate file?

Reply:
I think this is fine as is

Reply:
It's a bit big so consider creating two child components.

ExampleHeader which contains the code below until the map.
ExampleItem(?) which contains a single item. That would solve the div/React Fragment key issue